server/models/Stakeholder.js

```diff
@@ -1,513 +1,258 @@
+            type: entity.type === 'O' ? 'Organization' : 'Person',
+            org_name: entity.name,
+            first_name: entity.first_name,
+            last_name: entity.last_name,
+            email: entity.email,
+            phone: entity.phone,
+            logo_url: entity.logo_url,
+            map: entity.map_name,
+            website: entity.website,
+          };
+
+          const stakeholder =
+            await this._stakeholderRepository.createStakeholder(stakeholderObj);
+
+          if (i > 0) {
+            // if there are relations, create links
+            await this.createRelation(org_id, {
+              type: 'children',
+              data: stakeholder,
+            });
+          }
+        }
       });
+      await Promise.all(updates);
+    }
+
+    return org_id;
+  }
 
-      const children = await repo.getChildren(stakeholder);
+  async getUUID(org_id) {
+    const orgId = +org_id;
+    // get organization from old entity table
+    return orgId ? this.getUUIDbyId(orgId) : org_id;
+  }
 
-      stakeholder.children = children.map((child) => {
-        child.parents = [{ ...stakeholder }];
-        child.children = [];
-        return child;
-      });
+  async getRelationTrees(stakeholders) {
+    return Promise.all(
+      stakeholders.map(async (stakeholder) => {
+        const stakeholderCopy = { ...stakeholder };
+        const parents = await this._stakeholderRepository.getParents(
+          stakeholderCopy,
+        );
+        // don't want to keep getting all of the parents and children recursively, but do want to
+        // include the current stakeholder as parent/child
+
+        stakeholderCopy.parents = parents.map((parent) => {
+          const parentCopy = { ...parent };
+          parentCopy.parents = [];
+          parentCopy.children = [{ ...stakeholderCopy }];
+          return parentCopy;
+        });
 
-      return stakeholder;
-    }),
-  );
-
-const getAllStakeholders =
-  (repo) =>
-  async (
-    { filter: { where, order, limit, offset }, ...idFilters } = undefined,
-    url,
-  ) => {
-    let filter = {};
-    filter = FilterCriteria({ ...idFilters, ...where });
-
-    const options = {
-      ...QueryOptions({ limit, offset, ...order }),
-    };
+        const children = await this._stakeholderRepository.getChildren(
+          stakeholderCopy,
+        );
 
-    const { next, prev } = makeNextPrevUrls(url, filter, options);
+        stakeholderCopy.children = children.map((child) => {
+          const childCopy = { ...child };
+          childCopy.parents = [{ ...stakeholderCopy }];
+          childCopy.children = [];
+          return childCopy;
+        });
 
-    // get organization from old entity table, enter org id to insert it and it's children
-    // await getUUID(repo, 1);
+        return stakeholderCopy;
+      }),
+    );
+  }
 
+  async getAllStakeholders(filter, limitOptions) {
     let dbStakeholders;
     let count;
 
     if (Object.keys(filter).length > 0) {
```

Comment:
The filter is no longer required when I update the client to match this api so let's check for the filter before checking it's length  `filter && Object.keys(filter).length > 0`